corehq/motech/repeaters/models.py

```diff
@@ -1390,6 +1390,9 @@ def iter_partition(self, start, partition, total_partitions):
                 "id__gt": result[-1].id,
             }
 
+    def get_domains_having_records(self):
+        return self.order_by().values_list("domain", flat=True).distinct()
```

Comment:
nit: `get_domains_with_records` sounds more natural, but this is still understandable.

Reply:
11f430ca02a2192e09a43d12dfee0ed93ba345dd